agent/src/test/java/com/microsoft/applicationinsights/agent/internal/agent/ClassInstrumentationDataTest.java

```diff
@@ -28,10 +28,10 @@
 
 public final class ClassInstrumentationDataTest {
     private final static String MOCK_CLASS_NAME = "ClassName";
-    private final static String MOCK_METHOD = "Method";
-    private final static String MOCK_SIGNATURE_1 = "Signature1";
-    private final static String MOCK_SIGNATURE_2 = "Signature2";
-    private final static String MOCK_SIGNATURE_3 = "Signature3";
+//    private final static String MOCK_METHOD = "Method";
```

Comment:
remove instead of commenting. BTW, why commenting?

Reply:
Commenting because this signature was used in tests which were relying on getDecisionMethod() however it turns out that there is no such method in ClassInstrumentationData class.

Reply:
May be if there is an idea to get this tests running in future, that was the reason.

Reply:
What happens if you uncomment these tests?

Reply:
Please address these tests. If they are failing, they should be updated to reflect the new functionality. If they are no longer valid, they should be removed (and explain why this is the case).

Reply:
@littleaj all the asserts within this tests are commented because there is a method which they intended to write is missing. Therefore there is no use to create unnecessary objects during test run time. It will add overhead only.

Reply:
@dhaval24 as discussed, let's just revert this to the master copy and we'll come back to it later. Then, we'll figure out if it needs to be removed or modified so it works.